wpilibc/src/main/native/cpp/controller/RamseteController.cpp

```diff
@@ -68,3 +72,5 @@ ChassisSpeeds RamseteController::Calculate(
   return Calculate(currentPose, desiredState.pose, desiredState.velocity,
                    desiredState.velocity * desiredState.curvature);
 }
+
+void RamseteController::SetEnabled(bool enabled) { m_enabled = true; }
```

Comment:
Should be `m_enabled = enabled`

Reply:
Whoops... I was apparently more tired than I thought when I did this.